packages/datatrak-web/src/layout/UserMenu/ProjectSelectModal.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { Dialog, Paper } from '@material-ui/core';
+import { ProjectSelectForm } from '../../features';
+
+const Wrapper = styled(Paper)`
+  padding: 1.5rem 2.5rem 1.25rem;
+  max-width: none;
+  width: 48rem;
+`;
+
+interface ModalProps {
+  open: boolean;
+  onClose: () => void;
+  projectId?: string;
+}
+
+export const ProjectSelectModal = ({ open, onClose, projectId }: ModalProps) => {
+  return (
+    <Dialog open={open} onClose={onClose} PaperComponent={Wrapper} disablePortal>
```

Comment:
This might not have been merged in when you made this, but there is now a reusable modal component you can use

Reply:
Thanks but since i don't want the heading of the close button and have different padding I might just keep this as is